jobserver/models.py

```diff
@@ -321,6 +321,55 @@ def status(self):
         return "unknown"
 
 
+class Membership(models.Model):
+    project = models.ForeignKey(
+        "Project",
+        on_delete=models.CASCADE,
+        related_name="members",
+    )
+    user = models.ForeignKey(
+        "User",
+        on_delete=models.CASCADE,
+        related_name="memberships",
+    )
+
+    class Meta:
+        unique_together = ["project", "user"]
+
+    def __str__(self):
+        return f"{self.user.username} member of {self.project.name}"
+
+
+class Org(models.Model):
+    name = models.TextField(unique=True)
+
+    class Meta:
+        verbose_name = "Organisation"
+
+    def __str__(self):
+        return self.name
+
+
+class Project(models.Model):
+    org = models.ForeignKey(
+        "Org",
+        on_delete=models.CASCADE,
+        related_name="projects",
+    )
+
+    name = models.TextField(unique=True)
+    email = models.TextField()
+    project_lead = models.TextField()
```

Comment:
Should this be an FK to a `User`?

Reply:
Not yet, we don't have a way to onboard Users without giving them access to run Jobs.  It's possible it will be in the future but hard to say at this point.